src/webrtc/call.ts

```diff
@@ -1809,8 +1809,17 @@ export class MatrixCall extends EventEmitter {
         }
     }
 
-    private async placeCallWithConstraints(constraints: MediaStreamConstraints): Promise<void> {
-        logger.log("Getting user media with constraints", constraints);
+    /**
+     * Place a call to this room.
+     * @throws if you have not specified a listener for 'error' events.
+     * @throws if have passed audio=false.
+     */
+    public async placeCall(audio: boolean, video: boolean): Promise<void> {
+        logger.debug(`placeCall audio=${audio} video=${video}`);
+        if (!audio) {
+            throw new Error("You CANNOT start a call without audio");
```

Comment:
hmm.. why would it be a boolean option then?

Reply:
Because this should be possible in the future and I'd like to avoid another breaking change. In general, I feel like this will make things smoother